_topic_maps/_topic_map.yml

```diff
@@ -174,6 +170,15 @@ Topics:
     File: using-http2-gRPC
   - Name: Using Serving with OpenShift ingress sharding
     File: using-serving-with-ingress-sharding
+- Name: HTTP configuration
+  Dir: http-configuration
+  Topics:
+  - Name: Global HTTPS redirection
+    File: https-redirect-global
+  - Name: HTTPS redirection per service
+    File: https-redirect-per-service
+  - Name: Full duplex support for HTTP/1
+    File: http1-full-duplex-support
```

Comment:
Does this need to be a full subdirectory? Especially considering each of the 3 assemblies only have 1 module in each (with no TOCs).

Could this instead just be a single new "HTTP configuration" assembly that `include::`s the 3 modules (the 2 old ones and the 1 new one)? I feel like that would still satisfy the Jira ticket's request to "create another section".

Reply:
Yes, it does need to be a full subdirectory as each assemblies are going to expand by the upcoming next release and will contain the multiple modules. That is why I created like that. :)